Makefile

```diff
@@ -0,0 +1,113 @@
+SHELL=bash
+SOURCE_DIR = $(shell pwd)
+BIN_DIR = ${SOURCE_DIR}/bin
+COMPOSER = composer
+CI_DIR = ${BIN_DIR}/tool-php-ci
+AWS_SHARED_CREDENTIALS_FILE ?= $(HOME)/.aws/credentials
+
+define printSection
+	@printf "\033[36m\n==================================================\n\033[0m"
+	@printf "\033[36m $1 \033[0m"
+	@printf "\033[36m\n==================================================\n\033[0m"
+endef
+define replaceDotenv
+	test -f .env.dev.local || cp .env.dev .env.dev.local
+	TEMPFILE="$(shell mktemp)" && \
+	sed -e 's@^$1=.*@$1=$2@' .env.dev.local > "$$TEMPFILE" && \
+	mv "$$TEMPFILE" .env.dev.local
+endef
+
+.PHONY: all
+all: install quality test test-dependencies
+
+.PHONY: ci
+ci: quality test test-dependencies
+
+.PHONY: install
+install: clean-vendor composer-install
+
+.PHONY: quality
+quality: git-commit-checker cs-ci
+
+.PHONY: test
+test: atoum
+
+.PHONY: test-dependencies
+test-dependencies: sf-security-checker composer-source-checker
+
+# Coding Style
+
+.PHONY: cs
+cs:
+	${BIN_DIR}/php-cs-fixer fix --dry-run --stop-on-violation --diff
+
+.PHONY: cs-fix
+cs-fix:
+	${BIN_DIR}/php-cs-fixer fix
+
+.PHONY: cs-ci
+cs-ci:
+	${BIN_DIR}/php-cs-fixer fix --ansi --dry-run --using-cache=no --verbose
+
+#COMPOSER
+
+.PHONY: clean-vendor
+clean-vendor:
+	$(call printSection,CLEAN-VENDOR)
+	rm -rf ${SOURCE_DIR}/vendor
+
+.PHONY: composer-install
+composer-install: ${SOURCE_DIR}/vendor/composer/installed.json
+
+${SOURCE_DIR}/vendor/composer/installed.json:
+	$(call printSection,COMPOSER INSTALL)
+	$(COMPOSER) --no-interaction install --ansi --no-progress --prefer-dist
+	$(call replaceDotenv,AWS_SHARED_CREDENTIALS_FILE,$(AWS_SHARED_CREDENTIALS_FILE))
+	$(call replaceDotenv,AWS_ROLE_ARN,$(shell aws configure get role_arn --profile 6cloud-dev))
+
+# CI TOOLS
+
+.PHONY: sf-security-checker
+sf-security-checker: ${CI_DIR}
+	$(call printSection,COMPOSER SECURITY CHECKER)
+	${CI_DIR}/security-checker.phar security:check --ansi
```

Comment:
this project does not contain any `security-checker.phar`, I sugget to simply remove this target